lib/pharos/phase_manager.rb

```diff
@@ -64,9 +64,7 @@ def prepare_phase(phase_class, host, ssh: false, **options)
     def apply(phase_class, hosts, parallel: false, **options)
       phases = hosts.map { |host| prepare_phase(phase_class, host, **options) }
 
-      run(phases, parallel: parallel) do |phase|
-        phase.call
-      end
+      run(phases, parallel: parallel, &:call)
```

Comment:
Looks funny that way. Maybe the runner should do the `.call`ing.

Reply:
Blame `rubocop`

```
lib/pharos/phase_manager.rb:67:39: C: Style/SymbolProc: Pass &:call as an argument to run instead of a block.
      run(phases, parallel: parallel) do |phase| ...
```